web/src/hotkey.js

```diff
@@ -971,6 +971,18 @@ export function process_hotkey(e, hotkey) {
         case "all_messages":
             browser_history.go_to_location("#all_messages");
             return true;
+        case "toggle_topic_visibility_policy":
+            if (recent_view_ui.is_in_focus()) {
+                const recent_msg = recent_view_ui.get_focused_row_message();
+                if (recent_msg !== undefined) {
+                    user_topics_ui.toggle_topic_visibility_policy(recent_msg);
+                    return true;
+                }
+            }
+            if (inbox_ui.is_in_focus()) {
+                inbox_ui.toggle_topic_visibility_policy();
+                return true;
+            }
```

Comment:
This looks correct except it should `return false` in the case that view in question is in focus, but we didn't take any action. I think you can do that for inbox by writing `return inbox_ui.toggle_topic_visibility_policy();` and making that function have the appropriate return value; and for the recent view it's even simpler.